### PR TITLE
Normalize boundaries kind

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/tilezen/vector-datasource/issues/841

@nvkelso, @zerebubuth could you review please?

### CODE REVIEW

File: yaml/boundaries.yaml [hunk 1]
```diff
@@ -1,75 +1,79 @@
...
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: {col: admin_level}}
     table: osm
   - filter:
       admin_level: ['6', '8']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 10
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: {col: admin_level}}
```
Comment (nvkelso) [inline yaml/boundaries.yaml]: Is this transformed later on? Should we set it explicitly here to a value like `3`?

File: yaml/boundaries.yaml [hunk 2]
```diff
@@ -1,75 +1,79 @@
 filters:
+
+  # osm
   - filter:
       admin_level: ['2', '4']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 8
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: {col: admin_level}}
```
Comment (nvkelso) [inline yaml/boundaries.yaml]: Is this transformed later on? Should we set it explicitly here to a value like `3`?

Comment (rmarianski) [inline yaml/boundaries.yaml]: We used to have a transform in python, but that got removed when we moved to the yaml syntax. I updated this to set the value explicitly to 3 in 4958518.

File: yaml/boundaries.yaml [hunk 3]
```diff
@@ -1,75 +1,79 @@
 filters:
+
+  # osm
   - filter:
       admin_level: ['2', '4']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 8
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: '3'}
```
Comment (nvkelso) [inline yaml/boundaries.yaml]: Were we sending thru `2` or `4` or `6` or `8` before depending on the input?

Comment (rmarianski) [inline yaml/boundaries.yaml]: 2 or 4 for the above, and 6 or 8 for the below, although I couldn't find any actual data that matched the below.

Comment (rmarianski) [inline yaml/boundaries.yaml]: Updated to use the other naming scheme for aboriginal lands, namely `boundary=aboriginal_lands` in 17c3774.

**Comment by nvkelso** [comment]
(So merge the two filters, so all are result is min_zoom of 8.)

**Comment by rmarianski** [comment]
Merged in 6c32f2c.

**Pull request merged**